tests/modules/nf-core/verifybamid/verifybamid/nextflow.config

```diff
@@ -0,0 +1,15 @@
+// Including niche test data
+def test_niche_data_dir = "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/delete_me"
+params {
+    niche_test {
+        'verifybamid' {
+            verifybamid_test_bam = "${test_niche_data_dir}/verifybamid/verifybam_bam.bam"
+        }
+    }
+}
```

Comment:
Can we add a test for this?

Reply:
yeah I'm not sure what's going on here. You don't need to make your own data_dir nor a list for a `delete_me` file - you just directly supply the rawlink to `file()` in the ftest itself.